hypothesis-python/src/hypothesis/extra/pandas/impl.py

```diff
@@ -126,6 +126,8 @@ def convert_element(value):
             if is_na_dtype and value is None:
                 return None
             name = f"draw({prefix}elements)"
+            if dtype.kind == "O":
+                return value  # for objects, just use the object other numpy might convert it
```

Comment:
I'm not quite clear on what this comment means - make it a line or two above the return?